src/boot/export-vars.c

```diff
@@ -51,4 +52,10 @@ void export_common_variables(EFI_LOADED_IMAGE_PROTOCOL *loaded_image) {
                 s = xasprintf("UEFI %u.%02u", ST->Hdr.Revision >> 16, ST->Hdr.Revision & 0xffff);
                 efivar_set_str16(MAKE_GUID_PTR(LOADER), u"LoaderFirmwareType", s, 0);
         }
+
+        /* ditto for LoaderTpm2ActivePcrBanks */
+        if (efivar_get_raw(MAKE_GUID_PTR(LOADER), u"LoaderTpm2ActivePcrBanks", NULL, NULL) != EFI_SUCCESS) {
```

Comment:
hmm, not a fan of binary efi vars. we so far for numeric values formatted them as strings. I think we should do that too here.

i.e. take inspirtion from sysfs, where attributes tend to be formatted in decimal strings too always, even if bitmasks.